validation/custom_constraint.rst

```diff
@@ -12,20 +12,43 @@ alphanumeric characters.
 Creating the Constraint Class
 -----------------------------
 
-First you need to create a Constraint class and extend :class:`Symfony\\Component\\Validator\\Constraint`::
+First you need to create a Constraint class and extend :class:`Symfony\\Component\\Validator\\Constraint`:
 
-    // src/Validator/ContainsAlphanumeric.php
-    namespace App\Validator;
+.. versionadded:: 5.2
```

Comment:
This should be moved below the configuration block, thanks

Reply:
Done @OskarStark